packages/ckeditor5-font/tests/fontcommand.js

```diff
@@ -224,16 +224,13 @@ describe( 'FontCommand', () => {
 			);
 		} );
 
-		it( 'should use parent batch for storing undo steps', () => {
+		it( 'should use provided batch', () => {
 			setData( model, '<paragraph>a[bc<$text font="foo">fo]obar</$text>xyz</paragraph>' );
+			const batch = model.createBatch();
+			const spy = sinon.spy( model, 'enqueueChange' );
 
-			model.change( writer => {
-				expect( writer.batch.operations.length ).to.equal( 0 );
-				command.execute( { value: 'foo' } );
-				expect( writer.batch.operations.length ).to.equal( 1 );
-			} );
-
-			expect( getData( model ) ).to.equal( '<paragraph>a[<$text font="foo">bcfo]obar</$text>xyz</paragraph>' );
```

Comment:
I think having both tests (the old one and new one) makes sense - depending on whether you've passed the batch as a parameter.